src/tools/spectest-interp.cc

```diff
@@ -71,6 +71,7 @@ static void ParseOptions(int argc, char** argv) {
     s_log_stream = FileStream::CreateStdout();
   });
   parser.AddHelpOption();
+  parser.AddVersionOption();
```

Comment:
Since we are adding this to all tools, why not just call this in the OptionParser constructor?  

As a followup (to keep this change small) we could move AddHelpOption to the constructor too?

Reply:
Both updated in 87c82544f5552e416c264ac348c743876c5309ea